spec/duration.html

```diff
@@ -570,6 +570,11 @@ <h1>Properties of Temporal.Duration Instances</h1>
       Temporal.Duration instances are ordinary objects that inherit properties from the %Temporal.Duration.prototype% intrinsic object.
       Temporal.Duration instances are initially created with the internal slots described in <emu-xref href="#table-internal-slots-of-temporalduration-instances"></emu-xref>.
     </p>
+    <emu-note>
+      For an integer _x_ stored in an internal slot of this object, it must be true that ℝ(𝔽(_x_)) = _x_.
+      In other words, _x_ must be an integer that can be represented by a Number value.
```

Comment:
doesn’t this mean we can’t get MAX_SAFE_INTEGER+1, for example?

Reply:
That's right, the feedback from implementors that prompted this change is to avoid exactly that: having to keep track of a hidden BigInt value for the fields that could be observable. I'll make this clearer in the commit message and PR description.